Assignments/task-1/src/question/view.rs

```diff
@@ -0,0 +1,46 @@
+pub extern crate mysql;
+
+/// Structure 'Emp' having 2 fields
+///
+/// #Fields
+///
+/// emp_id: Is of i32 type
+/// emp_name: Is of String type
+pub struct Emp {
+    emp_id: i32,
+    emp_name: String,
+}
+
+/// Function 'view_db_table' prints the contents of the table in database
+///
+/// #Arguments
+///
+/// query_test: Is if type String
+/// link: Is of type String
+///
+/// #Return
+///
+/// Returns Option<()>
+pub fn view_db_table(query_test: String, link: String) -> Option<()> {
```

Comment:
Please incorporate proper error handling and refactor naming conventions, documentation, and return type.
Tyr to ignore explicit panics.